src/components/product-card/ProductCard.jsx

```diff
@@ -101,17 +107,41 @@ export default function ProductCard ({
           </span>
         </div>
         <div className="card-vt-btn mt-5 ">
-          <a href="#" className="button button-primary button-text-icon ">
+          <button className="button button-primary button-text-icon ">
             <span>
               <i className="fas fa-shopping-cart"></i>
               Add to Cart
             </span>
-          </a>
-          <a href="#" className="button btn-outline button-secondary ">
-            <span>
-              <i className="far fa-heart"></i>
-            </span>
-          </a>
+          </button>
+          {wishState.wishlist.find((prod) => prod._id === _id) ? (
```

Comment:
You can extract out this logic as a separate common function  to check for both cart and wishlist by passing parameters accordingly

Reply:
Thanks Ravi, will note and Refactor at end!